diagnostic_rt_qpcr/libraries/diagnosticrtqpcrcompositions/source.rb

```diff
@@ -7,16 +7,55 @@
 #
 # @author Devin Strickland <strcklnd@uw.edu>
 module DiagnosticRTqPCRCompositions
-  PRIMER_MIX = 'Primer/Probe Mix'
+  PRIMER_PROBE_MIX = 'Primer/Probe Mix' # Should converge with PRIMER_PROBE_MIX
+      # in PCRCOmpositionDefinitions
   TEMPLATE = 'Template'
 
+  # Initialize all `PCRComposition`s for each operation stripwell
+  #
+  # @param operations [OperationList]
+  # @return [void]
+  def build_stripwell_master_mix_compositions(operations:)
+    operations.each do |op|
+      primer_mixes = []
+      op.input_array(PRIMER_PROBE_MIX).each do |fv|
+        primer_mixes += fv.collection.parts
+      end
+
+      compositions = []
+
+      primer_mixes.each do |primer_mix|
+        composition = build_modified_master_mix_composition(
+          primer_mix: primer_mix,
+          program_name: op.temporary[:options][:program_name]
+        )
+        compositions.append(composition)
+      end
+
+      op.temporary[:compositions] = compositions
+    end
+  end
+
+  # Initialize a `PCRComposition` for a given primer mix and program
+  # stripwell
+  #
+  # @param primer_mix [Item]
+  # @param program_name [String]
+  # @return [PCRComposition]
+  def build_modified_master_mix_composition(primer_mix:, program_name:)
```

Comment:
Are we actually adding the primer/probe mix or is it part of the dried down reagent?

Reply:
I'm worried that the PCRComposition isn't doing any work here because the composition is already made. I may have led you astray.

Reply:
The primer/probe is part of the dried down reagent.   Not sure if PCRComposition is doing much other than tracking provenance in a standard way.

Reply:
PCRComposition is really designed for tracking what you do when you're making the composition. The provenance methods in the other protocols are designed to use that info for deciding what to put in the provenance, but it isn't necessary to use that approach. It seems like it may be inappropriate here because you aren't really making anything. Sorry I didn't realize that sooner.

Reply:
In this case, i think the provenance should point to the dried reagent stripwell part, which is a sample with a name like "Modified CDC Master Mix with N1 Primer/Probe".